### ISSUE TITLE
Use multiple Where clauses when using fetch()

### ISSUE THREAD

**Issue by bhapps** [ImangazalievM/ReActiveAndroid#43]
ReActiveAndroid Version:
latest

Bug or Feature Request:
Use multiple Where clauses when using` fetch()`

Description:
There is still an issue where I wish to return all (or paginated) data filtered by the query using `fetch() `which returns the error of `Cannot bind argument at index 2 because the index is out of range. The statement has 1 parameters`

All the documentation advises only of using `fetchSingle() `when using multiple Where clauses
I wish not to return only the first hit, but all (or some) data items.

- Maybe `fetch()` has not been implemented for using multiple Where clauses
- `fetch() `with multiple clauses would require `List<>` return type

**Comment by bhapps** [comment]
Ok funny thing as per below, two args passed through and got results from `fetch()` ...

```
List<MyTableClass> MyTableClassList = Select.from(MyTableClass.class).where(
"id BETWEEN ? AND ?", 
firstId, 
lastId)
    .orderBy("id " + defaultSortBy)
    .limit(limit)
    .fetch();
```

however, using `LIKE '?&' `causes crash ...
Ill investigate further ..

**Comment by ImangazalievM** [comment]
@Bennith yeah, ReActiveAndroid has a problem with `LIKE` clause. I will try to catch and fix this bug.
